test/index.js

```diff
@@ -17,14 +17,16 @@ function functionallyIdentical(tester){
     };
 }
 
-function testExpression(name, expression, scope, expected, comparitor){
+function testExpression(name, expression, scope, expected, comparitor, only){
```

Comment:
Instead of passing a bool as 'only'. add an 'only' function onto testExpression so it can be called like:

```
testExpression.only(...)
```